### ISSUE TITLE
[WebGL2] An error occurred in PoseNet example when scale factor = 1

### ISSUE THREAD

**Issue by yl495** [intel/webml-polyfill#142]
**Test Env:** 
Chromium Version: nightly build (revision e7980af)
OS: Mac OS

**Expected Result:** 
PoseNet should execute successfully.  

**Actual Result:** 
An error occurred when scale factor = 1: 
```
Uncaught (in promise) Error: [Tensor] numSlices 17 > MAX_TEXTURE_IMAGE_UNITS 16
    at Tensor._createTextureSlices (webml-polyfill.js:1094)
    at Tensor.createGLTexture (webml-polyfill.js:1027)
    at Input.call (webml-polyfill.js:21503)
    at webml-polyfill.js:8170
    at Array.forEach (<anonymous>)
    at webml-polyfill.js:8158
    at new Promise (<anonymous>)
    at Model._execute (webml-polyfill.js:8150)
    at Model.execute (webml-polyfill.js:8120)
    at Execution._callee$ (webml-polyfill.js:7968)
``` 
**How to Reproduce:** 
1. Launch chromium, open PoseNet example and switch backend to WebGL2
2. Adjust scale factor in control interface to 1

**Comment by GreyZzzzzzXh** [comment]
```
    if (numSlices > webgl2.MAX_TEXTURE_IMAGE_UNITS) {	
      throw new Error(`[Tensor] numSlices ${numSlices} > MAX_TEXTURE_IMAGE_UNITS ${webgl2.MAX_TEXTURE_IMAGE_UNITS}`);	
    }
```
This check code makes no sense, _MAX_TEXTURE_IMAGE_UNITS_ is not used in the latest Keras.js.
Let's remove it, then everything will goes well.

**Comment by huningxin** [comment]
@GreyZzzzzzXh , thanks for the investigation! Could you please point out which commit in Keras.js removed this check and for what reason?

**Comment by GreyZzzzzzXh** [comment]
@huningxin , actually this check has never been used since the first support of the slicing operation, and I don't even find any history of the use of MAX_TEXTURE_IMAGE_UNITS.

This is the code that supports slice for the first time. 
[https://github.com/transcranial/keras-js/commit/64a16fbd5adb49b7a00dbe0cb348fd206638427c#diff-a797aba7c38597bce5bbe0d05a74a3ca](https://github.com/transcranial/keras-js/commit/64a16fbd5adb49b7a00dbe0cb348fd206638427c#diff-a797aba7c38597bce5bbe0d05a74a3ca)

**Comment by huningxin** [comment]
I recall there was some issues on smartphone. Are you able to verify whether it works on mobile browser?

**Comment by GreyZzzzzzXh** [comment]
@huningxin , it works well on smartphone now. Issue #182 can also be fixed by #215. Thank @BruceDai 's help!
